community-membership.md

```diff
@@ -297,5 +298,72 @@ with the project. When a contributor returns to being more active, they may be
 promoted back to their previous position at the discretion of the current
 maintainers following the process defined above.
 
+## Specification Sponsor
+
+Specification sponsors are trusted collaborators of the technical committee, and
+work to review, approve, and sponsor [opentelemetry-specification][] issues and
+PRs.
+
+Members are defined
+in [Community Members -> Specification and Protos](./community-members.md#specifications-and-proto)
+and are members of the `spec-sponsors` team (TODO: add link to github team when
+available). Specification sponsors SHOULD list
+their [areas of interest](./areas-of-interest.md).
+
+### Requirements
+
+Specification sponsors are trusted to drive specification work which can be
+complex and highly impactful to downstream language SIGs and to the
+OpenTelemetry project as a whole. Approvals from specification sponsors count
+towards the approvals required to merge specification PRs. Therefore, there is a
+high bar to becoming a specification sponsor.
+
+- Contributor to OpenTelemetry project for at least 1 year
+- Has been the reviewer or author (through the sponsorship of another
+  specification sponsor) of at least 10 substantial PRs to specification or
+  related repositories
+- Demonstrated sound technical judgement, and a deep understanding of the
+  specification and the OpenTelemetry project
+- Nominated by technical committee member (
+  see [becoming a specification sponsor](#becoming-a-specification-sponsor))
+
+### Responsibilities and privileges
+
+- Review, respond to, and approve issues / PRs when tagged or when within areas
+  of interest.
+- Sponsor issues, taking responsibility to be point on requirement gathering,
+  developing solutions and prototypes, writing PRs, and responding to /
+  incorporating PR feedback. This can be done by the sponsor themselves, or by
+  another collaborator who is sponsored. If sponsoring another collaborator, the
+  sponsor should work with them to ensure the work is consistent with existing
+  specification.
+- Demonstrate sound technical judgement.
+
+Sponsors who are unable to fulfill their responsibilities (e.g. no longer have
+time available to commit) are expected to be moved
+to [emeritus](#emeritus-maintainerapprovertriager) sponsors. The TC may also
+vote to move a sponsor to emeritus.
+
+### Becoming a Specification Sponsor
+
+Technical committee members nominate specification sponsors by opening a PR to
+the community repo to add the nominee to the specification sponsor list. The
+vote is officially started when a pull request is opened, and ends when the pull
+request is merged. The pull request may be merged when the following conditions
+are met:
+
+- The person being nominated has accepted the nomination by approving the pull
+  request.
+- All TC members have approved the pull request or a majority of TC members have
```

Comment:
Do we go with super majority? i.e. 2/3? Not a blocking concern though.

Reply:
This language came from the requirements to become a maintainer:

> Unless stated otherwise in a SIG charter ratified by the Technical Committee,
a new maintainer is elected by vote of the existing maintainers of the SIG.
The vote is officially started when a pull request to add a new maintainer
is opened, and ends when the pull request is merged. The pull request may be
merged when the following conditions are met:
> - The person being nominated has accepted the nomination by approving the pull request
> - All maintainers have approved the pull request OR a majority of maintainers
  have approved the pull request and no maintainer has objected by requesting
  changes on the pull request. In the case that all maintainers have not given
  approval, the pull request should stay open for a minimum of 5 days before merging.

The super majority requirement is the same as the requirement to add members to TC:

> A candidate can be elected to the TC by the super-majority vote (greater than two thirds) of the existing TC members.

So there's a question of whether we want spec sponsor election to be more like maintainer or more like TC. The unanimous approval or abstain requirement to become a maintainer may help ensure the maintainers work better together since members are only added who all agree or are neutral on. The 2/3 supermajority requirement allows members to be elected which some people actively disapprove of.

Hard to know which voting requirement will be more appropriate for this function. We can always start with the unanimous approve or abstain requirement and later relax to be 2/3 supermajority.

Reply:
In my opinion the function of this group is more similar to that of the technical committee than any maintainer group. Reasonable and limited disagreement is important to ensure the specification evolves carefully in the right direction and the 2/3 supermajority rule best reflects this.

Reply:
I'm don't see any strong opinions from TC members so I'll resolve this comment for now but we should come back to it if we run into issues.